tapchannel/aux_invoice_manager.go

```diff
@@ -121,6 +124,14 @@ func (s *AuxInvoiceManager) handleInvoiceAccept(_ context.Context,
 
 	// No custom record on the HTLC, so we have nothing to do.
 	if len(req.WireCustomRecords) == 0 {
+		// If there's no wire custom records and the invoice is an asset
+		// invoice do not settle the invoice.
+		//
+		// TODO(george): Strict-forwarding could be configurable?
```

Comment:
Not sure we need this TODO here. IMO if you want assets you create an asset invoice. If not, you create a normal one.

Reply:
This was done with this feature case in mind: I create an invoice of value X, and i may receive X via sats or via assets in any combination